### ISSUE TITLE
Command: validate

### ISSUE THREAD

**Issue by h3rald** [h3rald/glyph#37]
```
glyph validate
glyph validate links
glyph validate snippets
```

**Comment by h3rald** [comment]
Not necessary: validation is performed while compiling.

**Comment by h3rald** [comment]
Implemented validation.
- Closed by a84313442b0f50e49871980bfba557a3ce3f4b21.
- Closed by a84313442b0f50e49871980bfba557a3ce3f4b21.
- Closed by a84313442b0f50e49871980bfba557a3ce3f4b21.
